admin/static/css/keen-dashboards.css

```diff
@@ -96,3 +96,25 @@ body.application .return-link:hover,
   font-size: 12px;
   padding: 8px 10px 5px;
 }
+
+.keen-dataviz-table-dataset td{
+  min-width: 60px !important;
+}
+
+#dropdownMenuButton {
+  margin-top: 20px;
+  margin-right: 20px;
+}
+
+thead {
```

Comment:
Have you verified that none of the css you added affects any of the rest of the admin app?